dashboard/db/migrate/20160718080224_remove_user_id_from_sections.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUserIdFromSections < ActiveRecord::Migration
+  def up
+    remove_column :sections, :user_id
+  end
+
+  def down
+    add_column :sections, :user_id, :integer
+  end
```

Comment:
For simple migrations, we encourage (though not require) using the `change` method. For example, the above can be written more succinctly (including the existing nonnullness constraint) (barring any mistakes by me, I didn't test) as
```
def change
  alter_table(:sections) do
    remove_column :user_id, :integer, null: false
  end
end
```
See http://edgeguides.rubyonrails.org/active_record_migrations.html#changing-tables for more information.